### PR TITLE
KOGITO-2252 Restore Travel Agency v01

### PR DESCRIPTION
**Pull request by evacchi**
Restore this version of Travel Agency (the simplest one, with no persistence, no Visa App)
  Commit # https://github.com/kiegroup/kogito-examples/commit/a106b7447ec5c5ade5a972954efc73cb842bbc11
  Update README accordingly.

https://issues.redhat.com/browse/KOGITO-2252

### CODE REVIEW

**Comment by radtriste** [comment]
@evacchi you just moved the current to `extended` folder and put the version 01 into `basic`?

**Comment by radtriste** [comment]
Please be aware that this PR will cause our BDD tests to fail.
I will open a dependent PR for this to update the those.

**Comment by radtriste** [comment]
Opened https://issues.redhat.com/browse/KOGITO-2412

**Comment by MarianMacik** [comment]
By looking at the tree here:
https://github.com/evacchi/kogito-examples/tree/KOGITO-2252/kogito-travel-agency
It seems that `docker-compose`, `operator` and `scripts` directories were not moved under the extended directory, does it?
Can you move everything under the `extended` directory, please?

**Comment by evacchi** [comment]
@radtriste yes that is all, thanks
@MarianMacik right, will do EDIT: done, thanks

File: kogito-travel-agency/README.md [hunk 1]
```diff
@@ -14,8 +14,11 @@ NOTE: Look into individual versions to learn more on how to run it or what does
 Each version is based on previous one so it allows people perform the implementation at their own pace
 and to start any any place.
 
-[Base travel agency service with persistence](travels) - deals with all the basic steps to book flight and hotel.
-Consists of single service that has both business logic and decision logic included. Preserves data between service restarts and
+[Base travel agency service](basic) - deals with all the basic steps to book flight and hotel.
+Consists of single service that has both business logic and decision logic included. 
+
+[Travel agency and visa services with persistence](travels) - deals with all the basic steps to book flight and hotel.
```
Comment (radtriste) [inline kogito-travel-agency/README.md]:
```suggestion
[Travel agency and visa services with persistence](extended) - deals with all the basic steps to book flight and hotel.
```

**Comment by radtriste** [comment]
@evacchi Jenkinsfile has changed, please rebase your PR :)

File: kogito-travel-agency/README.md (hunk 1 above)
Comment (evacchi) [reply kogito-travel-agency/README.md]: done! thx

**Comment by evacchi** [comment]
ah that's why it was red :)

File: kogito-travel-agency/README.md [hunk 2]
```diff
@@ -14,8 +14,11 @@ NOTE: Look into individual versions to learn more on how to run it or what does
 Each version is based on previous one so it allows people perform the implementation at their own pace
 and to start any any place.
 
-[Base travel agency service with persistence](travels) - deals with all the basic steps to book flight and hotel.
-Consists of single service that has both business logic and decision logic included. Preserves data between service restarts and
+[Base travel agency service](basic) - deals with all the basic steps to book flight and hotel.
+Consists of single service that has both business logic and decision logic included. 
+
+[Travel agency and visa services with persistence](extended) - deals with all the basic steps to book flight and hotel.
+Consists of two services that has both business logic and decision logic included. Preserves data between service restarts and
```
Comment (MarianMacik) [inline kogito-travel-agency/README.md]: that have?

File: kogito-travel-agency/README.md [hunk 3]
```diff
@@ -14,8 +14,11 @@ NOTE: Look into individual versions to learn more on how to run it or what does
...
 requires Infinispan server to be available.
```
Comment (MarianMacik) [inline kogito-travel-agency/README.md]: Also would be good to have this on one line also in the md.file.

**Review by MarianMacik — APPROVED** [review]
Just minor things, otherwise looks good.

**Review by radtriste — APPROVED** [review]
lgtm

**Review by MarianMacik — APPROVED** [review]

**Comment by radtriste** [comment]
@cristianonicolai @evacchi This PR is green: https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/blue/organizations/jenkins/KIE%2Fkogito%2Fghprb-webhooks%2Fkogito-examples/detail/kogito-examples/20/pipeline

and so can be merged :)
I will merge the Operator one once this one is merged

**Comment by evacchi** [comment]
@radtriste right! merged

**Pull request merged**